services/api-server/tests/unit/pact_broker/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
if everything is well defined and you do not use local imports, there is no need to add `__`init__.py` in tests